### DISCUSSION TITLE
How to create a dynamic rounte URL

### DISCUSSION THREAD

**Discussion by ParveeenRao** [mockoon/mockoon#1213] [General]
Hi, 

I am configuring a post request which accepts customerid in the path parameter. 
e.g https:test.com/v1/12212/payment where 12212 is the customerid(integer). 

So Instead of configuring the URL as 'v1/12212/payment' in Mockoon, I want the request to execute for all customers if the rule matches something like 'v1/{customerid}/payment'. Is it possible to configure it in this way?

Thanks,
Parveen

**Comment by 255kb** [comment]
Hi

You will find the relevant documentation [here](https://mockoon.com/docs/latest/api-endpoints/routing/#route-parameters).
